task-management/src/main/webapp/vue-app/tasks-management/components/ProjectTasks/TasksViewDashboard.vue

```diff
@@ -211,6 +211,11 @@
           this.tasksList = this.tasksList.filter((t) => t.id !== event.detail);
         }
       });
+      this.$root.$on('update-task-completed', (event) => {
+        if (event) {
+          this.tasksList = this.tasksList.filter((t) => t.id !== event.id);
```

Comment:
the same, I think you need to check if Show completed tasks is enabled or not to execute this filter